gradle/versions.gradle

```diff
@@ -23,11 +23,11 @@ ext {
     mapbox_navigation = '0.42.5'
     mapbox_sdk = '9.0.0'
 
-    mannodermaus_junit5_version = '1.4.2.1'
+    mannodermaus_junit5_version = '1.6.0.0'
```

Comment:
@kiryldz this plugin caused `javac blabla` issues

Reply:
Indeed that helps, thanks!